lib/client/api.go

```diff
@@ -2390,14 +2390,8 @@ func (tc *TeleportClient) LogoutAll() error {
 	return nil
 }
 
-// Login logs the user into a Teleport cluster by talking to a Teleport proxy.
-//
-// The returned Key should typically be passed to ActivateKey in order to
-// update local agent state.
-//
-func (tc *TeleportClient) Login(ctx context.Context) (*Key, error) {
-	// Ping the endpoint to see if it's up and find the type of authentication
-	// supported.
+// PingAndShowMOTD pings the Teleport Proxy and displays the MOTD if it's available.
```

Comment:
nit: I don't abbreviation in the function name, but I'd assume that comment explains what MOTD is. Without any context this can be not obvious.
```suggestion
// PingAndShowMOTD pings the Teleport Proxy and displays the Message Of The Day if it's available.
```